packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.js

```diff
@@ -226,6 +226,8 @@ export default class QuanticCategoryFacet extends LightningElement {
       facetId: this.facet.state.facetId,
       element: this.template.host,
     });
+
+    this.toggleVisibility = this.toggleFacetVisibility.bind(this);
```

Comment:
Yes, I was using the function `toggleFacetVisibility` not `toggleFacetVisibility`. The names are quit confusing. I tried to put `this.toggleFacetVisibility = this.toggleFacetVisibility.bind(this)`, but it would break the page and I am not sure why.